reconcile/test/test_jump_host.py

```diff
@@ -0,0 +1,117 @@
+import os
+from typing import (
+    Any,
+    Optional,
+)
+from unittest.mock import create_autospec
+
+import pytest
+
+from reconcile.utils import gql
+from reconcile.utils.jump_host import (
+    JumpHostBase,
+    JumphostParameters,
+    JumpHostSSH,
+)
+
+EXPECTED_USER = "test-user"
+EXPECTED_HOSTNAME = "test"
+EXPECTED_KNOWN_HOSTS_PATH = "path/to/known-hosts"
+EXPECTED_KNOWN_HOSTS_CONTENT = "known-hosts-file-content"
+
+
+@pytest.mark.parametrize(
+    "parameters,expected_port",
```

Comment:
Please add a space after the `,` 🙇

Reply:
Good catch. Fixed it here and in another test file now too.